app/client/widgets/ChoiceToken.ts

```diff
@@ -34,10 +34,22 @@ export function choiceToken(
   options: IChoiceTokenOptions,
   ...args: DomElementArg[]
 ): DomContents {
+  return cssChoiceToken(choiceTokenDomArgs(label, options, ...args));
+}
+
+/**
+ * Exposes the choiceToken dom args outside of cssChoiceToken to allow
+ * easy usage of them with TokenField#renderToken, that has its own wrapper dom el.
+ */
+export function choiceTokenDomArgs(
+  label: DomElementArg,
+  options: IChoiceTokenOptions,
+  ...args: DomElementArg[]
```

Comment:
I think it's better to omit this `...args` argument -- the caller can simply add extra args to the DOM after the result of this call, e.g.
```
cssChoiceToken(choiceTokenDomArgs(label, options), ...args);
```